scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -197,7 +198,8 @@ class ScanCode(
         }
     }
 
-    override val scannerVersion = "3.0.2"
+    // Strip the 'rc2' suffix to avoid "SemverException: Invalid version (no patch version): 3.2.1rc2".
+    override val scannerVersion = SCANNER_VERSION.removeSuffix("rc2")
```

Comment:
I had no better idea to solve this. Anyone?

Reply:
Shouldn't inserting a dash before "rc2" work?

Reply:
No idea, need to investigate it.

Reply:
Applied an idea analog to yours.